site/content/en/docs/formats/video.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Video'
+linkTitle: 'Video'
+description: ''
+---
+
+## Format specification
+There are many kinds of video extensions as listed up
+[here](https://github.com/openvinotoolkit/datumaro/blob/develop/datumaro/plugins/data_formats/video.py).
+
+Datumaro can import a video into video frames by adjusting the start frame, end frame,
+and step size. Furthermore, with a `video_keyframes` format, Datumaro can extract
+keyframes by comparing zero-mean normalized cross correlation (ZNCC) metric between
+successive frames as following [here](https://www.sciencedirect.com/science/article/pii/S1047320312000223).
+Plus, Datumaro provides the options for choosing image extension and name patterns
+for efficient data management from multiple videos.
+
+## Import video
+
+A Datumaro project with a video frames can be created
+in the following way:
+
+```
+datum create
+datum import -f video_frames <path_to_video>
+```
+
+Load video through the Python API:
+
+```python
+import datumaro as dm
+
+dataset = dm.Dataset.import_from('<path_to_video>', format='video_frames')
+```
+
+Datumaro has few import options for `video_frames` format, to apply them
+use the `--` after the main command argument.
+
+`video_frames` import options:
+- `-i, --input-path` (string) - Path to the video file
+- `-o, --output-dir` (string) - Output directory. By default, a subdirectory
+  in the current directory is used
+- `--overwrite` - Allows overwriting existing files in the output directory,
+  when it is not empty
+- `-p, --name-pattern` (string) - Name pattern for the produced
+  images (default: `%06d`)
+- `-s, --step` (integer) - Frame step (default: 1)
+- `-b, --start-frame` (integer) - Starting frame (default: 0)
+- `-e, --end-frame` (integer) - Finishing frame (default: none)
+- `-x, --image-ext` (string) Output image extension (default: `.jpg`)
+- `-h, --help` - Print the help message and exit
+
+Usage:
+
+``` bash
+datum import -f video_frames [-h] SRC_PATH [-o DST_DIR] [--overwrite]
+  [-n NAME_PATTERN] [-s STEP] [-b START_FRAME] [-e END_FRAME] [-x IMAGE_EXT]
+```
```

Comment:
It seems that this is different with what I run in `1.1.0rc1`
```console
(datumaro) vinnamki@jisunkwo-desktop:~/datumaro$ datum import -f video_frames -- -h
usage: video_frames [-h] [--subset SUBSET] [-p NAME_PATTERN] [-s STEP] [-b START_FRAME] [-e END_FRAME]

Reads video frames as a dataset.

optional arguments:
  -h, --help            show this help message and exit
  --subset SUBSET       The name of the subset for the produced dataset items (default: none)
  -p NAME_PATTERN, --name-pattern NAME_PATTERN
                        The name pattern for the produced dataset items (default: %06d).
  -s STEP, --step STEP  Frame step (default: 1)
  -b START_FRAME, --start-frame START_FRAME
                        Starting frame (default: 0)
  -e END_FRAME, --end-frame END_FRAME
                        Finishing frame (default: None)
```

```console
(datumaro) vinnamki@jisunkwo-desktop:~/datumaro$ datum import -f video_frames -h
usage: datum import [-h] [-n NAME] -f FORMAT [-r RPATH] [--no-check] [-p PROJECT_DIR] url ...

Imports a data source to a project. A data source is a dataset in a supported format (check 'formats' section below).

Currently, only local paths to sources are supported.
During importing, a source is copied into the project.

Formats:
Datasets come in a wide variety of formats. Each dataset format defines its own data structure and rules on how to interpret the data. Check the user manual for the list of
supported formats, examples and documentation.
The list of supported formats can be extended by plugins. Check the "plugins" section of the developer guide for information about plugin implementation.

Each dataset format has its own import options, which are passed after the '--' separator (see examples), pass '-- -h' for more info.

Builtin formats: ade20k2017, ade20k2020, align_celeba, anns_to_labels, ava, bbox_values_decrement, boxes_to_masks, brats, brats_numpy, camvid, celeba, cifar, cityscapes, coco,
coco_captions, coco_image_info, coco_instances, coco_labels, coco_panoptic, coco_person_keypoints, coco_stuff, common_semantic_segmentation,
common_semantic_segmentation_with_subset_dirs, common_super_resolution, crop_covered_segments, cvat, datumaro, datumaro_binary, icdar_text_localization,
icdar_text_segmentation, icdar_word_recognition, id_from_image_name, image_dir, image_zip, imagenet, imagenet_txt, imagenet_with_subset_dirs, item, kinetics, kitti,
kitti_detection, kitti_raw, kitti_segmentation, label_me, label_random_sampler, lfw, map_subsets, mapillary_vistas, mapillary_vistas_instances, mapillary_vistas_panoptic,
market1501, mars, masks_to_polygons, merge_instance_segments, merge_tile, mnist, mnist_csv, mot_seq, mots, mots_png_extractor, mpii, mpii_json, mvtec, mvtec_classification,
mvtec_detection, mvtec_segmentation, ndr, nyu_depth_v2, open_images, polygons_to_masks, project_infos, project_labels, random_sampler, random_split, reindex,
relevancy_sampler, remap_labels, remove_annotations, remove_attributes, remove_items, rename, resize, shapes_to_boxes, sly_pointcloud, split, synthia, tf_detection_api, tile,
vgg_face2, video_frames, video_keyframes, voc, voc_action, voc_classification, voc_detection, voc_layout, voc_segmentation, vott_csv, vott_json, wider_face, yolo, yolo_loose,
yolo_strict, yolo_ultralytics

Examples:
- Add a local directory with a VOC-like dataset:
  datum import -f voc path/to/voc

- Add a directory with a COCO dataset, use only a specific file:
  datum import -f coco_instances -r anns/train.json path/to/coco

- Add a local file with CVAT annotations, call it 'mysource'
    to the project in a specific place:
  datum import -f cvat -n mysource -p project/path/ path/to/cvat.xml

positional arguments:
  url                   URL to the source dataset. A path to a file or directory
  extra_args            Additional arguments for extractor (pass '-- -h' for help). Must be specified after the main command arguments and after the '--' separator

optional arguments:
  -h, --help            show this help message and exit
  -n NAME, --name NAME  Name of the new source (default: generate automatically)
  -f FORMAT, --format FORMAT
                        Source dataset format
  -r RPATH, --path RPATH
                        A path relative to URL to the source data. Useful to specify a path to subset, subtask, or a specific file in URL.
  --no-check            Don't try to read the source after importing
  -p PROJECT_DIR, --project PROJECT_DIR
                        Directory of the project to operate on (default: current dir)
```

Reply:
Thank you for the correction. At that time, I started from copying https://github.com/openvinotoolkit/datumaro/blob/develop/site/content/en/docs/user-manual/command-reference/util.md. And this might be old version too. I will update this together.